Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/OrientationCubeController.cs

```diff
@@ -20,5 +20,17 @@ public void UpdateOrientation(Transform rootBP, Transform target)
             //UPDATE ORIENTATION CUBE POS & ROT
             transform.SetPositionAndRotation(rootBP.position, lookRot);
         }
+
+        public void UpdateOrientation(Vector3 pos, Vector3 dirVector)
```

Comment:
The walker agent uses the new method. The other environments will be updated to use the new method in a follow up PR

Reply:
I think the code would be easier to read if `UpdateOrientation(Transform rootBP, Transform target)` called `UpdateOrientation(Vector3 pos, Vector3 dirVector)`. This is to avoid code duplication.

Reply:
I reset this to Master. changing it would require updating other agents scripts which i think should be done in a future PR